From 1c9ef24a311c2cbd300dc26fbde1bac28ff75227 Mon Sep 17 00:00:00 2001
From: Chao Sun <sunchao@apache.org>
Date: Wed, 17 May 2017 14:22:02 -0700
Subject: [PATCH 1131/1164] CDH-54015: HIVE-16700: Log ZK discovery info
 (hostname & port) for HTTP mode when connection
 is established (Chao Sun, reviewed by Xuefu
 Zhang)

(cherry picked from commit 89699ef4be80150cc31f89eefaf9d3db20e30e6e)

Change-Id: I5438e79fbd87b1490acaa0333a3eeecb7da69762
---
 .../java/org/apache/hive/jdbc/HiveConnection.java  |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/jdbc/src/java/org/apache/hive/jdbc/HiveConnection.java b/jdbc/src/java/org/apache/hive/jdbc/HiveConnection.java
index 1eb3d39..b03f690 100644
--- a/jdbc/src/java/org/apache/hive/jdbc/HiveConnection.java
+++ b/jdbc/src/java/org/apache/hive/jdbc/HiveConnection.java
@@ -200,8 +200,8 @@ private void openTransport() throws SQLException {
         transport = isHttpTransportMode() ? createHttpTransport() : createBinaryTransport();
         if (!transport.isOpen()) {
           transport.open();
-          logZkDiscoveryMessage("Connected to " + connParams.getHost() + ":" + connParams.getPort());
         }
+        logZkDiscoveryMessage("Connected to " + connParams.getHost() + ":" + connParams.getPort());
         break;
       } catch (TTransportException e) {
         // We'll retry till we exhaust all HiveServer2 nodes from ZooKeeper
-- 
1.7.9.5

